### PR TITLE
Added jp2a tool.

### PR DESCRIPTION
**Pull request by fredlebel**
A small utility that converts JPG images to ASCII.

### CODE REVIEW

File: pkgs/applications/misc/jp2a/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, libjpeg }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.6";
+  name = "jp2a-${version}";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/project/jp2a/jp2a/${version}/${name}.tar.gz";
```
Comment (cillianderoiste) [inline pkgs/applications/misc/jp2a/default.nix]: Please use the mirror url for sourceforge.

File: pkgs/applications/misc/jp2a/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    sha256 = "076frk3pa16s4r1b10zgy81vdlz0385zh3ykbnkaij25jn5aqc09";
+  };
+
+  makeFlags = "PREFIX=$(out)";
+
+  buildInputs = [ libjpeg ];
+
+  meta = with stdenv.lib; {
```
Comment (cillianderoiste) [inline pkgs/applications/misc/jp2a/default.nix]: Would you also like to add yourself as the maintainer?

File: pkgs/applications/misc/jp2a/default.nix (hunk 1 above)
Comment (fredlebel) [inline pkgs/applications/misc/jp2a/default.nix]: Changed to use mirror.

File: pkgs/applications/misc/jp2a/default.nix (hunk 2 above)
Comment (fredlebel) [inline pkgs/applications/misc/jp2a/default.nix]: Maybe when I have more free time.  I would not be able to take on that commitment.

**Comment by cillianderoiste** [comment]
Thanks @fredlebel!

**Pull request merged**